src/service.ts

```diff
@@ -47,6 +54,30 @@ export class PisaService {
             next();
         });
 
+        // rate limits
+        if (config && config.rateGlobal) {
```

Comment:
The middleware section is quite large now, lets move all app.use() methods into a new function applyMiddleware(app) just so that the constructor is a bit easier to read.

Reply:
Done.